src/test/java/seedu/recipe/model/recipe/exceptions/RecipeDurationInvalidArgumentLengthExceptionTest.java

```diff
@@ -0,0 +1,21 @@
+package seedu.recipe.model.recipe.exceptions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.model.recipe.RecipeDuration;
+
+public class RecipeDurationInvalidArgumentLengthExceptionTest {
+
+    @Test
+    public void validRecipeDurationInvalidArgumentLengthException() {
+        Exception exception = assertThrows(RecipeDurationInvalidArgumentLengthException.class, () ->
+                RecipeDuration.of("15"));
+        String expectedMessage = "An argument list of invalid length was passed for a Recipe Duration."
+                + "\nEnsure it is of the following format: '{number OR decimal} {duration}; "
+                + "\nExample: `1 minute`, `1.5 hours`";
```

Comment:
make this a `static final` constant at the start of the test class so its easier to modify